Makefile

```diff
@@ -44,32 +63,54 @@ lint: ## Run golangci-lint
 verify: tidy generate format lint ## Verify the current code generation and lint
 	git diff --exit-code
 
-KUBERNETES_VERSION=v0.22.2
 COMBO_VERSION=$(shell git describe || echo $(DEFAULT_VERSION))
+BUILD_FLAGS=CGO_ENABLED=0
```

Comment:
I'm assuming this is to make using `Docker.local` easier?

Reply:
I more wanted to expose a valve for overriding all of the build flags associated with building the binary to give the user some agency over that.

Reply:
My two cents... I might be in the minority with this opinion, but for Go projects, I see the Makefile as the way to provide the opinionated "best" set of flags to compile/test/run the project.

Since its Go, a developer can pretty easily invoke their own `go build` command with whatever flags they like without us having to clutter our Makefile up to support it.